R/create_default_parameters.R

```diff
@@ -602,6 +602,57 @@ create_default_recruitment <- function(
 #' @seealso
 #' * [create_default_parameters()]
 #' @export
+#' @examples
+#' \dontrun{
+#' # Prepare the package data for being used in a FIMS model
+#' data("data1")
+#' fims_frame <- FIMSFrame(data1)
+#'
+#' # Define the fleet specifications 
+#' fleet1 <- survey1 <- list(
+#'   selectivity = list(form = "LogisticSelectivity"),
+#'   data_distribution = c(
+#'     Index = "DlnormDistribution",
+#'     AgeComp = "DmultinomDistribution"
+#'   )
+#' )
+#' fleet2 <- list(
+#'   selectivity = list(form = "DoubleLogisticSelectivity"),
+#'   data_distribution = c(
+#'     Index = "DlnormDistribution",
+#'     AgeComp = "DmultinomDistribution",
+#'     LengthComp = "DmultinomDistribution"
+#'   )
+#' )
```

Comment:
fleet2 does not actually exist in fims_frame so when I try to run this example I get
```
Error in `create_default_parameters()`:
ℹ The name of the fleets for selectivity settings must match the fleet names present in the `data`.
✖ The following 1 fleet name is missing from the data: fleet2.
```
I think we can delete these lines and simplify line 631 to just have fleet1 and survey1.

Reply:
Sounds good. I updated the examples for `create_default_parameters()` based on the suggestions for fleet2. For the examples for `update_parameters()`, I removed fleet2 entirely and focused on different ways to checking parameter lengths.